cmd/cluster-capi-operator/main.go

```diff
@@ -226,6 +227,15 @@ func main() {
 		setupReconcilers(mgr, infra, platform, &gcpv1.GCPCluster{}, containerImages, applyClient, apiextensionsClient, *managedNamespace)
 		setupWebhooks(mgr)
 	case configv1.AzurePlatformType:
+		azureCloudEnvironment := getAzureCloudEnvironment(infra.Status.PlatformStatus)
+		if azureCloudEnvironment == configv1.AzureStackCloud {
+			unsupportedPlatform = true
+
+			klog.Infof("detected Azure Cloud Environment %q on platform %q is not supported, skipping capi controllers setup", azureCloudEnvironment, platform)
```

Comment:
Should log lines start with a capital?
```suggestion
			klog.Infof("Detected Azure Cloud Environment %q on platform %q is not supported, skipping capi controllers setup", azureCloudEnvironment, platform)
```

Reply:
SGTM